shared/AppInsightsCore/Tests/Selenium/CorePerfCheck.Tests.ts

```diff
@@ -280,7 +280,11 @@ export class CorePerfCheckTests extends AITestClass {
                         Assert.equal(iterations * objectFields * optDuration.attempts, checks, "Make sure we hit all of them");
     
                         this._checkRun(baseDuration, optDuration);
+                    }, (reason) => {
+                        Assert.ok(false, "Promise rejected - " + reason);
```

Comment:
Under some cases there was an UnhandledRejectionException being thrown and logged (failing the tests) with no details on why.

So still failing the tests, but causing information to be logged.